### ISSUE TITLE
Javier Cano

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#187]
How he got started?
- Worked in-house at Walmart (Q/A)
-Worked in San Fran (immigrated software to new servers)
- From their moved to a company in Texas, developed (Angular/ React, backend was Java based)
- Has a pretty extensive background in multiple areas

Why are you passionate about coding?

What is your go to coding language? Why?

Depending on size of project - PHP with react or
larger would use node, JS
Main is JS + React, knows PHP as well
React Native? not really has used flutter a little bit?
What are you looking for in your next job roles?

He's really interested in the whole project that we are working on on what he has seen, wants his code to be in a final product, he enjoys seeing his code in the final stages, being part of a team
How do you feel most comfortable working?

Has all he needs in his set up - 2 monitors a keyboard
-Not having a ton of meetings, some are obvious necessary, but dev takes time
Likes his own schedule
How comfortable would you be in a virtual working session? That would be fine, being able to ask questions would make things flow better.
What's the largest team who've worked in?

As devs, 550 people working for one of his companies, working as a backend engineer, developing some services in a small squad.
-Management? It was complex, worked in a squad of 8-10 communicated with the other squads - had a tech lead and a product owner
Best management of a project that you have experienced? Why did it work so well?

He really enjoys using SCRUM, the daily meetings and the tasks for each sprint laid out. Being able to communicate daily.
Did you track your efficiencies - devs in general struggle with due dates, but it gets better as you move along, looking at effort is important in understanding how much time things will take
Career Goals?

Currently continue building dev skills and being involved in complex things
Would want to manage and mentor some other members
Management Experence?

Mentored some of the jr. devs?

**Comment by aflies1** [comment]
https://drive.google.com/file/d/1ZA8NAuOsjVoHxeMIwjf_ZZCXda1T9Esq/view?usp=sharing

**Comment by ldaudet** [comment]
JAVIER EMMANUEL FABELA CANO - 9/19/22 9:30


His english is a bit of a problem

His got 7 to 8 years of experience. He started working on backend only. He specialized on the Java backend. He then moved to QA, where he learned about appium and selenium framework for automated testing. He worked on testing for the walmart application. 

He then switched to a development team and become the manager for the front end team. The frotn end was in .Net and Java. Lots of microservices patterns. 

He never finished it because he moved to omnitracks in texas, a logistic company for providing parts for trucks. He was a intermediate dev for the backend, all in Java based, micro services patterns. He got all the apps and services the company had into a single API. Some of the services where in COBOL, some in .NET. He used GRPC frameworks to connect all the services. He used Groovy and Jenkins for pipelines and jobs.

At Globant he is using AWS and Google cloud at the same time. He is using Maven as his basis. 

He is most familiar with AWS and Azure. He got a call for this position, but he was not really looking for a position, he was most interested as to what was available in the market.

TL;DR: his English is just a bit low, making communication a bit laborious. He also isn't as good a match on the tech stack side of things having spent quite a lot of time in Java and QA dev. Finally, he seemed to not be really interested in the position and more looking to see what's out there. He said he liked the work he was currently doing. I think he isn't a perfect fit for the team
